pydicom/valuerep.py

```diff
@@ -43,6 +43,58 @@
 # (the component separator '=' is handled separately)
 PN_DELIMS = {0x5e}
 
+FIXED_VALUE_VRS = ("AS", "DA")
+
+MAX_VALUE_LEN = {
+    "AE": 16,
+    "AS": 4,
+    "CS": 16,
+    "DA": 8,
+    "DS": 16,
+    "DT": 26,
+    "IS": 12,
+    "LO": 64,
+    "LT": 10240,
+    "PN": 196,
```

Comment:
Also, 64 chars max per component, is 64 * 5 = 320?

Reply:
Yes, I think PN has to been separately (e.g. check the length of each component), and the check can be removed here then.